code/modules/cm_marines/equipment/sentries.dm

```diff
@@ -477,6 +490,36 @@
 				visible_message("\icon[src] <span class='notice'>The [name] powers down and goes silent.</span>")
 				update_icon()
 
+		if("toggle_alert")
+			if(!alerts_on)
+				user.visible_message("<span class='notice'>[user] activates [src]'s alert notifications.</span>",
+				"<span class='notice'>You activate [src]'s alert notifications.</span>")
+				visible_message("\icon[src] <span class='notice'>The [name] buzzes in a monotone voice: 'Alert notification system initiated'.</span>'")
+				alerts_on = TRUE
+				update_icon()
+			else
+				alerts_on = FALSE
```

Comment:
This could be condensed with `alerts_on = !alerts_on` and `alerts_on ? this : that`